Client/Frontend/Strings.swift

```diff
@@ -244,6 +244,15 @@ extension String {
                 comment: "Accessibility label for a credit card list item in autofill settings screen. The first parameter is the credit card type (e.g. Visa). The second parameter is the last 4 digits of the credit card. The third parameter is the name of the credit card holder. The fourth and fifth parameters are the month and year of the credit card's expiration date.")
         }
 
+        // Displaying a credit card
+        public struct DisplayCard {
+            public static let ExpiresLabel = MZLocalizedString(
+                key: "CreditCard.EditCard.RevealLabel.v114",
```

Comment:
This key is already in used, I suggest:
```suggestion
                key: "CreditCard.DisplayCard.ExpiresLabel.v115",
```